portfolio/src/custom.scss

```diff
@@ -1,9 +0,0 @@
-// Override default variables before the import
-$body-bg: #000;
-// Import Bootstrap and its default variables
-@import '~bootstrap/scss/bootstrap.scss';
-@import url('https://fonts.googleapis.com/css?family=Source+Code+Pro|Montserrat|Playfair+Display|Questrial');
```

Comment:
does this not work with regular css?? P sure you can set font faces with regular css so you dont need an scss compiler... Maybe amplify is weirder than I thought